### PR TITLE
add title prop

### PR DESCRIPTION
**Pull request by juanmendez90**

### CODE REVIEW

**Review by danivm — APPROVED** [review]

File: src/sui-autocompleted/index.jsx [hunk 1]
```diff
@@ -205,11 +206,13 @@ Autocompleted.propTypes = {
...
-  suggests: PropTypes.array.isRequired
+  suggests: PropTypes.array.isRequired,
+  title: PropTypes.string
 }
 
 Autocompleted.defaultProps = {
   initialValue: '',
   selectFirstByDefault: true,
-  focus: false
+  focus: false,
```
Comment (AitorRodriguez990) [inline src/sui-autocompleted/index.jsx:214]: Order properties please 🙏

File: src/sui-autocompleted/index.jsx [hunk 2]
```diff
@@ -167,7 +167,7 @@ export default class Autocompleted extends Component {
   }
 
   render () {
-    const { placeholder, handleFocus, handleBlur } = this.props
+    const { placeholder, handleFocus, handleBlur, title } = this.props
```
Comment (AitorRodriguez990) [inline src/sui-autocompleted/index.jsx]: Order properties, please 🙏

**Review by AitorRodriguez990 — APPROVED** [review]

File: src/sui-autocompleted/index.jsx [hunk 3]
```diff
@@ -205,11 +206,13 @@ Autocompleted.propTypes = {
...
+  suggests: PropTypes.array.isRequired,
+  title: PropTypes.string
 }
 
 Autocompleted.defaultProps = {
+  focus: false,
   initialValue: '',
   selectFirstByDefault: true,
-  focus: false
+  title: undefined
```
Comment (midudev) [inline src/sui-autocompleted/index.jsx]: If it's `undefined`, then defaultProp is not needed.

Comment (juanmendez90) [reply src/sui-autocompleted/index.jsx]: fixed

**Pull request merged**